examples/prostate/configs/prostate_central/config/config_fed_server.json

```diff
@@ -0,0 +1,95 @@
+{
+  "format_version": 2,
+  "min_clients": 1,
+  "num_rounds": 100,
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "name": "PTFileModelPersistor",
+      "args": {
+      "model": {
+            "path": "monai.networks.nets.unet.UNet",
+            "args": {
+              "dimensions": 3,
+              "in_channels": 1,
+              "out_channels": 1,
+              "channels": [16, 32, 64, 128, 256],
+              "strides": [2, 2, 2, 2],
+              "num_res_units": 2
+            }
+          }
+      }
+    },
+    {
+      "id": "shareable_generator",
+      "name": "FullModelShareableGenerator",
+      "args": {}
+    },
+    {
+      "id": "aggregator",
+      "name": "InTimeAccumulateWeightedAggregator",
+      "args": {
+        "aggregation_weights": {
+          "client_MSD": 1.0,
+          "client_I2CVB": 1.0,
+          "client_NCI_ISBI_3T": 1.0,
+          "client_NCI_ISBI_Dx": 1.0
+        }
+      }
+    },
+    {
+      "id": "model_selector",
+      "name": "IntimeModelSelectionHandler",
+      "args": {}
+    },
+    {
+      "id": "model_locator",
+      "path": "pt.pt_model_locator.PTModelLocator",
+      "args": {
+        "model_dir": "app_server/models"
```

Comment:
This is not good since it uses the internal impl knowledge of the PTFileModelPersistor that the models are kept with "app_server" folder.

Please use the built-in model locator.